library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -20,18 +20,31 @@ PlatformBridgeFilterConfig::PlatformBridgeFilterConfig(
           Api::External::retrieveApi(proto_config.platform_filter_name()))) {}
 
 PlatformBridgeFilter::PlatformBridgeFilter(PlatformBridgeFilterConfigSharedPtr config)
-    : platform_filter_(config->platform_filter()) {}
+    : platform_filter_(*config->platform_filter()) {
+  platform_filter_.instance_context = platform_filter_.init_filter(platform_filter_.static_context);
+
+  ASSERT(platform_filter_.instance_context, "init_filter unsuccessful");
+}
+
+void PlatformBridgeFilter::onDestroy() {
+  ASSERT(platform_filter_.instance_context, "init_filter must be called initially");
```

Comment:
```suggestion
  ASSERT(platform_filter_.instance_context, "expected initialized filter context");
```

Reply:
Same below. I think this is a bit clearer since technically `init_filter` can fail per the above assert

Reply:
Sounds good.